test/unit/spec/loadingWidgetSpec.js

```diff
@@ -1,52 +1,66 @@
-define(['jquery', 'widgets/loadingWidget'], ($, LoadingWidget) => {
+define(['jquery', 'widgets/loadingWidget', 'text!/kbase_templates/loading.html'], (
+    $,
+    LoadingWidget,
+    LoadingTemplate
+) => {
     'use strict';
 
+    const templateHtml = LoadingTemplate.replace(
+        '{{ static_url("kbase/images/kbase_animated_logo.gif") }}',
+        '/narrative/static/kbase/images/kbase_animated_logo.gif'
+    );
```

Comment:
pull in the template for the loading section -- otherwise any DOM checks will be pointless as the widget relies on a certain DOM structure being in place.